frontend/src/components/HorizontalLinearStepper.jsx

```diff
@@ -178,11 +178,12 @@ export default function HorizontalLinearStepper(data) {
       id:2, 
       comp: <MissionConfiguration droneArrayJson={setMainJson} id="Drones" mainJsonValue={mainJson} windowHeight={windowSize.current[1]}/>
     },
-    {
```

Comment:
@Hoshi-Okami You do not need to comment Test configs in this component array. Changes at new line [250](https://github.com/UAVLab-SLU/DRV_public/pull/37/files#diff-94814afc4c91d23b49ddaaa3c395e3bb99480e847ebcc6b8ced1dac7c0722b90R250) are enough. You can uncomment it and push it again.